handlers_global.go

```diff
@@ -0,0 +1,76 @@
+package veneur
+
+import (
+	"compress/zlib"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+
+	"golang.org/x/net/context"
+)
+
+type contextHandler func(c context.Context, w http.ResponseWriter, r *http.Request)
+
+func (ch contextHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	// nil context is bad form, but since we don't use it, it's a quick hack
+	// to allow us to write tests before we upgrade to Go 1.7 Context type
+	// and the newer goji
+	// TODO(aditya) actually update this
+	ch(nil, w, r)
+}
+
+// handleImport generates the handler that responds to POST requests submitting
+// metrics to the global veneur instance.
+func handleImport(s *Server) http.Handler {
```

Comment:
ah okay, when you put it like that it makes more sense - the fact that this is a free function is just a testing detail